reviews/models/mixins.py

```diff
@@ -182,17 +195,75 @@ def resubmission_allowed(self, ev):
+        context['notify_comment'] = self.get_info().get('notify_comment')
+        context['is_rejected'] = self.get_info().get('is_rejected')
+        reviews_email.send(context=context)
 
     def notify_reject(self, ev):
-        # TODO email node admins (MOD-53)
-        pass
+        context = self.get_context()
+        context['template'] = 'reviews_submission_status',
+        context['is_pre_moderation'] = self.get_info().get('is_pre_moderation')
+        context['notify_comment'] = self.get_info().get('notify_comment')
+        context['is_rejected'] = self.get_info().get('is_rejected')
+        reviews_email.send(context=context)
 
     def notify_edit_comment(self, ev):
-        # TODO email node admins (MOD-53)
-        pass
+        context = self.get_context()
+        context['template'] = 'reviews_update_comment'
+        reviews_email.send(context=context)
+
+    def get_info(self):
+        return dict(
+            is_rejected=self.action.to_state == workflow.States.REJECTED.value,
+            notify_comment=not (self.reviewable.provider.reviews_comments_private and not self.action.comment),
+            is_pre_moderation=self.reviewable.provider.reviews_workflow == workflow.Workflows.PRE_MODERATION.value
+        )
+
+    def get_context(self):
+        return dict(
+            creator=self.reviewable.node.creator,
+            email_recipients=[contributor._id for contributor in self.reviewable.node.contributors],
+            settings=settings.DOMAIN,
+            node=self.reviewable.node,
+            reviewable_title=self.reviewable.node.title,
+            reviewable_url=self.reviewable.absolute_url,
+            provider=self.reviewable.provider,
+            provider_url=self.reviewable.provider.external_url if self.reviewable.provider.external_url is not None else settings.DOMAIN + '/preprints' + self.reviewable.provider._id
+        )
+
+    @reviews_email.connect
+    def reviews_notification(self, context):
+        timestamp = timezone.now()
+        event_type = utils.find_subscription_type('global_reviews')
+        template = ''.join(context.get('template')) + '.txt.mako'
+        for user_id in context.get('email_recipients'):
+            user = OSFUser.load(user_id)
+            subscriptions = get_user_subscriptions(user, event_type)
+            if user == context.get('creator'):
+                context['is_creator'] = True
+            else:
+                context['is_creator'] = False
+            for notification_type in subscriptions:
+                if (notification_type != 'none' and subscriptions[notification_type] and user_id in subscriptions[notification_type]):
+                    node_lineage_ids = get_node_lineage(context.get('node')) if context.get('node') else []
+                    context['user'] = user
+                    message = mails.render_message(template, **context)
+                    digest = NotificationDigest(
```

Comment:
You'd mentioned a user setting to choose whether to receive a digest email or each one individually -- is that now gone? Are we going with all digests, all the time?

Reply:
All records go to notificationdigest all them time, then based on the type which can be email_digest, or email_transactional the celery task will pick daily or immediately